extensions/wikia/AssetsManager/AssetsManagerServer.class.php

```diff
@@ -107,6 +107,10 @@ public static function serve(WebRequest $request) {
 			$headers['X-Pass-Cache-Control'] = 'public, max-age=' . $cacheDuration['client'];
 		}
 
+		if ( $type === 'sasses' && strpos( $request->getRequestURL(), urlencode( 'crossorigin=anonymous' ) ) !== false ) {
```

Comment:
Now `crossorigin=anonymous` can be added to any sass url (e.g. manually) and it'll be returned with additional header. Is it expected?